src/picai_prep/dcm2mha.py

```diff
@@ -165,35 +166,38 @@ class Case:
     pass
 
 
+@dataclass
 class Dicom2MHACase(Case):
-    settings: Dicom2MHASettings = Dicom2MHASettings({}, {})
+    input_dir: Path
+    patient_id: str
+    study_id: str
+    paths: List[PathLike]
+    settings: Dicom2MHASettings = Dicom2MHASettings({}, {})  # TODO: add factory or set to None->init later
```

Comment:
why allow None at all?

Reply:
Fair enough, removing the default parameter is easier as well